LEAF_Request_Portal/sources/FormWorkflow.php

```diff
@@ -1443,12 +1455,22 @@ public function handleEvents(int $workflowID, int $stepID, string $actionType, ?
 
                         $email = new Email();
 
-                        $title = strlen($requestRecords[0]['title']) > 45 ? substr($requestRecords[0]['title'], 0, 42) . '...' : $requestRecords[0]['title'];
-                        $truncatedTitle = trim(strip_tags(htmlspecialchars_decode($title, ENT_QUOTES | ENT_HTML5 )));
+                        $formType = trim(strip_tags(
+                            htmlspecialchars_decode($requestRecords[0]['categoryName'], ENT_QUOTES | ENT_HTML5)
+                        ));
+
+                        $fullTitle = trim(strip_tags(
+                            htmlspecialchars_decode($requestRecords[0]['title'], ENT_QUOTES | ENT_HTML5)
+                        ));
+                        if($requestRecords[0]['needToKnow'] === 1) {

```

Comment:
Little nervous here.  Are we sure that "needToKnow" is getting set as an int?  'Cause a string "1" will fail here.

Reply:
It is set to int in database, it should come back as int. I would also say we should parse it as an int for belt and suspenders